docs/source/llms/tracing/index.rst

```diff
@@ -0,0 +1,615 @@
+Tracing in MLflow
+=================
+
+.. note::
+    MLflow Tracing is currently in **Experimental Status** and is subject to change without deprecation warning or notification. 
+
+MLflow offers a number of different options to enable tracing of your GenAI applications. 
+
+- **Automated tracing with LangChain**: MLflow provides a fully automated integration with LangChain that can activate by simply enabling ``mlflow.langchain.autolog()``.
+- **Manual trace instrumentation with high-level fluent APIs**: Decorators, function wrappers and context managers via the fluent API allow you to add tracing functionality with minor code modifications.
+- **Low-level client APIs for tracing**: The MLflow client API provides a thread-safe way to handle trace implementations, even in aysnchronous modes of operation.
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    overview
```

Comment:
<img width="284" alt="Screenshot 2024-06-05 at 8 39 58" src="https://github.com/mlflow/mlflow/assets/31463517/ef3468e7-79c7-4e8d-81fd-f8a2f5088774">

nit: the concept page is listed in the tree twice, probably we can remove this one?